.circleci/config.yml

```diff
@@ -366,6 +451,21 @@ workflows:
               only:
                 - /.*-stable/
                 - master
+  deploy:
+    jobs:
+      - publish-npm
+    branches:
+      only:
+        - /.*-stable/
+        - master
```

Comment:
This will publish ALWAYS from `stable` regardless of the tests. We should either wait for other workflows programatically to complete (no API yet that I am aware that allows it - you can only queue up jobs within particular workflow) or make this job "manual", meaning that I have to go to CircleCI and manually click "Deploy" . I actually believe that would be better approach. Many times, CI builds fail because of Flow, flakiness or something. Having manual way to approve build regardless of the failure would make it easier to avoid such issues in the future.

Reply:
I'm leaning towards taking advantage of the manual approval, for the reasons mentioned. I need to look into who would have the necessary permission to approve the deploy to go out - I wouldn't want to hold you back from pushing out a release if this is only available to FB employees. (It's midnight here, I'll check up on this in the morning)

Reply:
Okay. I think write access might be needed. If I need to undergo special security constraints here or something (like use a separate account), happy to comply with it. I think given my past experience, ti would be great to not wait for FB since you guys are pretty busy anyway!